src/libraries/System.Private.CoreLib/src/System/Reflection/Emit/AssemblyBuilder.cs

```diff
@@ -23,6 +29,14 @@ public ModuleBuilder DefineDynamicModule(string name)
             return DefineDynamicModuleCore(name);
         }
 
+        /// <summary>
+        /// When overridden in a derived class, defines a dynamic module in this assembly.
+        /// </summary>
+        /// <param name="name">The name of the dynamic module.</param>
+        /// <remarks>
+        /// .NET core currently support only one module.
```

Comment:
Suggest changing
`.NET core currently support only one module.`
to
`.NET Core supports only one module.`

The "Core" terminology is useful here; I don't think there's a better way to say this, although I suppose we could say `.NET v8+ supports only one module.`